examples/python-kernel-wrapper/cython/kernel.pyx

```diff
@@ -0,0 +1,18 @@
+import cudf
+from cudf._lib.cpp.table.table_view cimport mutable_table_view
+from cudf._lib.table cimport Table
+from libcpp.string cimport string
+
+cdef extern from "src/kernel_wrapper.hpp":
+    cdef cppclass C_CudfWrapper "CudfWrapper":
+        C_CudfWrapper(mutable_table_view tbl)
+        void tenth_mm_to_inches(int column_index)
+
+cdef class CudfWrapper:
+    cdef C_CudfWrapper* gdf
+
+    def __cinit__(self, Table t):
+        self.gdf = new C_CudfWrapper(t.mutable_view())
+
+    def tenth_mm_to_inches(self, col_index):
+        self.gdf.tenth_mm_to_inches(col_index)
```

Comment:
Is there a reason the example is based around making a C++ class instead of just a free function? I think the example would be more concise and easier to follow if `tenth_mm_to_inches` was just a free function taking in a `cudf::(mutable_)table_view` as input and returning a `cudf::(mutable_)table_view` or nothing if it just modifies the table inplace.

Reply:
This is a good point. The cython will definitely be simpler, as will the C++/CUDA.